src/tools/dart-fix.md

```diff
@@ -1,16 +1,23 @@
 ---
 title: dart fix
-description: Help for bringing your code up-to-date.
+description: Command-line tool for applying analysis fixes and migrating API usages.
 toc: false
 ---
 
-The `dart fix` command (added in Dart 2.12)
+The `dart fix` command
 finds and fixes two types of issues:
 
-* Analysis issues that have associated automated fixes
+* Analysis issues identified by [`dart analyze`][]
+  that have associated automated fixes
   (sometimes called _quick-fixes_ or _code actions_)
+
+  To learn more about customizing the analysis issues identified,
+  see [Customizing static analysis](/guides/language/analysis-options).
 * Issues that have associated package API migration information
 
+  For information about how automated package API changes work,
+  see [Data driven fixes][] on the Flutter wiki.
```

Comment:
The most recent decision that I'm aware of is that we don't want to advertise the data-driven fixes support. We are supporting it for Flutter because they're already using it, but we're not supporting it for other users. Please remove this reference to the documentation.

Reply:
Should I not mention that `dart fix` will fix these at all or just not document how they work/how to add them?

Reply:
Just don't document the mechanism that makes them work. It's good to mention that it also helps migrate your code to new versions of the Dart and Flutter SDKs.

Reply:
Thanks for the context. Adjusting the documentation in https://github.com/dart-lang/site-www/pull/3987